lib/ipa_test_kit/igs/README.md

```diff
@@ -16,6 +16,3 @@ If your IG, or the specific version of the IG you want to test against, is not p
         ...
       end
 ```
-
-## 3. Inferno Validator UI
```

Comment:
I don't know why this is being removed. You should just update the commented out ig volume line for the old validator service as well.